### DISCUSSION TITLE
Assigning zone/boundary conditions in a flexible way

### DISCUSSION THREAD

**Discussion by cons013** [ansys/pyfluent#3558] [General]
Between fluent 23.2, 24.1 and 24.2 the names of certain keys in the dictionaries that describe zones were changed, making developing scripts somewhat hard to maintain. To make life easier, is there a way to change a solid/fluid zone or boundary condition without using the rigid dicionary structure? From one of the github examples:

`session.settings.setup.cell_zone_conditions.solid["disc2"] = {
    "solid_motion": {
        "solid_motion_zone_motion_function": "none",
        "solid_motion_axis_direction": [0, 1, 0],
        "solid_motion_axis_origin": [-0.035, -0.821, 0.045],
        "solid_motion_velocity": [0, 0, 0],
        "solid_omega": -15.79,
        "solid_relative_to_thread": "absolute",
        "enable": True,
    }
}`

While not written here, in 24.1 'source_terms' became 'sources', and there are other instances of this. From 23.2 to 24.1 I believe the structure of the dict itself even changed. It would be nice if there was a simple, list-based or similar way to change these properties, does this currently exist? 

Thanks

**Comment by ansjschuetz** [comment]
I may be mistaken, but cannot you write e.g....
`session.settings.setup.cell_zone_conditions.solid["disc2"].solid_motion.solid_omega = -7.89`
...to just make a "surgical" change...?
